### PR TITLE
Add missed links of commands

### PR DESCRIPTION
**Pull request by hzy001**
Signed-off-by: Hao Ziyu <haoziyu@qiyi.com>

### CODE REVIEW

File: services/src/main/java/io/druid/cli/CliMiddleManager.java [hunk 1]
```diff
@@ -58,7 +58,7 @@
  */
 @Command(
     name = "middleManager",
-    description = "Runs a Middle Manager, this is a \"task\" node used as part of the remote indexing service."
+    description = "Runs a Middle Manager, this is a \"task\" node used as part of the remote indexing service, see http://druid.io/docs/latest/Middlemanager.html for a description"
```
Comment (gianm) [inline services/src/main/java/io/druid/cli/CliMiddleManager.java]: http://druid.io/docs/latest/design/middlemanager.html is a more up-to-date link.

File: services/src/main/java/io/druid/cli/CliPeon.java [hunk 2]
```diff
@@ -106,7 +106,7 @@
 @Command(
     name = "peon",
     description = "Runs a Peon, this is an individual forked \"task\" used as part of the indexing service. "
-                  + "This should rarely, if ever, be used directly."
+                  + "This should rarely, if ever, be used directly. See http://druid.io/docs/latest/Peons.html for a description"
```
Comment (gianm) [inline services/src/main/java/io/druid/cli/CliPeon.java]: http://druid.io/docs/latest/design/peons.html is more up to date.

File: services/src/main/java/io/druid/cli/CliRouter.java [hunk 3]
```diff
@@ -65,7 +65,7 @@
  */
 @Command(
     name = "router",
-    description = "Experimental! Understands tiers and routes things to different brokers"
+    description = "Experimental! Understands tiers and routes things to different brokers, see http://druid.io/docs/latest/Router.html for a description"
```
Comment (gianm) [inline services/src/main/java/io/druid/cli/CliRouter.java]: http://druid.io/docs/latest/development/router.html is more up to date.

File: services/src/main/java/io/druid/cli/CliMiddleManager.java (hunk 1 above)
Comment (hzy001) [reply services/src/main/java/io/druid/cli/CliMiddleManager.java]: done

File: services/src/main/java/io/druid/cli/CliPeon.java (hunk 2 above)
Comment (hzy001) [reply services/src/main/java/io/druid/cli/CliPeon.java]: done

File: services/src/main/java/io/druid/cli/CliRouter.java (hunk 3 above)
Comment (hzy001) [reply services/src/main/java/io/druid/cli/CliRouter.java]: done

**Comment by hzy001** [comment]
Anyway, the original links can jump to the more up to date links.

**Pull request merged**